main.cpp

```diff
@@ -52,106 +52,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: slider
+//  action 1: the slider goes up and down
+slider.moveUpAndDown();
+//  action 2: the slider reutrn to previous position
+slider.undoPreviousPosition();
+//  action 3: the slider return to default position
+slider.defaultPosition();
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise